PIL/Image.py

```diff
@@ -667,6 +715,7 @@ def __setstate__(self, state):
         self.frombytes(data)
 
     def tobytes(self, encoder_name="raw", *args):
```

Comment:
Backwards compatibility. Plugins for PIL were added in the Python 1.6 era, IIRC.